api/insert/insert.go

```diff
@@ -0,0 +1,180 @@
+package insert
```

Comment:
I named this to be in parallel with `api/revoke`, but it makes sense to be more explicit about what it does.